src/test/java/com/hocs/test/pages/accordion/CaseDetailsAccordion.java

```diff
@@ -0,0 +1,153 @@
+    public WebElementFacade whenWasTheCorrespondenceSent;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Should the response be copied to Number 10?']/parent::span")
+    public WebElementFacade shouldTheResponseBeCopiedToNumber10;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'How was the correspondence received?']/parent::span")
+    public WebElementFacade howWasTheCorrespondenceReceived;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'When was the correspondence received?']/parent::span")
+    public WebElementFacade whenWasTheCorrespondenceReceived;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Private Office Team']/parent::span")
+    public WebElementFacade privateOfficeTeam;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Initial Draft Team']/parent::span")
+    public WebElementFacade initialDraftTeam;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Which is the primary topic?']/parent::span")
+    public WebElementFacade whichIsThePrimaryTopic;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'What sort of response is required?']/parent::span")
+    public WebElementFacade whatSortOfResponseIsRequired;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Do you want to QA this offline?']/parent::span")
+    public WebElementFacade doYouWantToQAThisOffline;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'How do you intend to respond?']/parent::span")
+    public WebElementFacade howDoYouIntendToRespond;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Which is the primary draft document?']/parent::span")
+    public WebElementFacade whichIsThePrimaryDraftDocument;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Can this correspondence be answered by your team?']/parent::span")
+    public WebElementFacade canThisCorrespondenceBeAnsweredByYourTeam;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Do you approve the response?']/parent::span")
+    public WebElementFacade doYouApproveTheResponse;
+
+    @FindBy(xpath = "//div[contains(@class ,'govuk-accordion__section--expanded')]/descendant::strong[text() = 'Are you able to dispatch this?']/parent::span")
+    public WebElementFacade areYouAbleToDispatchThis;
```

Comment:
These xpaths can probably be made much smaller

Reply:
I spent a while on them, and I couldnt find a way to make them more specific. 
They need the text from an element that cant be uniquely identified apart from its sibling element, and the sibling isnt always unique apart from being able to determin that its ancestor element is expanded.